npu.user.js

```diff
@@ -1205,6 +1438,10 @@ var npu = {
 			dec = tmp_arr.join("");
 			return dec;
 		},
+
+		isPassingGrade: function(gradeStr) {
+			return gradeStr != "Elégtelen";

```

Comment:
If we are doing this, then let's also add `nem felelt meg` and `fail` for the 3-grade evaluation and the English UI, just to be comprehensive.